src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedToArray = sourceString.split(';');
```

Comment:
The variable name 'convertedToArray' is not very descriptive. Consider a name that reflects the content of the array, such as 'styleDeclarations' or 'cssProperties'.